### DISCUSSION TITLE
Group Assignment #5

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#34] [Assignments]
#### Due 5/25

For [Group Assignment #5](https://albertkun.github.io/22S-ASIAAM-191A/assignments/week7/group_assignment/), create a rough draft of your website, include a `readme.md`, and be prepared to give a short 2-3 minute presentation with your group members. 

### Submission
Post the link to your GitHub repository and website below.

ex.
**Group Repo:** https://github.com/akochaphum/my-awesome-group
**Group website:** https://akochaphum.github.io/my-awesome-group/index.html

**Comment by ntran27** [comment]
Repo: https://github.com/ntran27/22s-asiaam-191a-group
Website: https://ntran27.github.io/22s-asiaam-191a-group/final/index.html

**Comment by joonyloony** [comment]
repo: https://github.com/ananyakodali17/tax-redistribution-asian-am-191a
website: https://joonyloony.github.io/22s-191a-assignments/week8/index.html

**Comment by ashley-lu** [comment]
Repo: https://github.com/shubhgupta2018/uclgay
Website: https://shubhgupta2018.github.io/uclgay/index.html

**Comment by kaittli** [comment]
repo: https://github.com/kaittli/aanhpi-healthcare
website: https://kaittli.github.io/aanhpi-healthcare/index.html

**Comment by codingdianat** [comment]
repo: https://github.com/stefuntran/AAPI-Discrimination
website: https://stefuntran.github.io/AAPI-Discrimination/

**Comment by daikis12** [comment]
repo: https://github.com/daikis12/group-8
website: https://daikis12.github.io/group-8/final/index.html

**Comment by bittertang** [comment]
repo: https://github.com/bittertang/cuddly-broccoli
website: https://bittertang.github.io/cuddly-broccoli/

**Comment by spolcharoen** [comment]
repo: https://github.com/penelopelamb/wasted-opportunities
site: https://penelopelamb.github.io/wasted-opportunities/index.html

**Reply by penelopelamb** [reply]
slayyyy

**Comment by emmashanson** [comment]
repo: https://github.com/emmashanson/asianam191Akristenemmakeri
site: https://emmashanson.github.io/asianam191Akristenemmakeri/final/index.html
